packages/app/src/debug/DebugResults.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div
+    class="flex gap-8px items-center"
+    data-cy="debug-results-holder"
+  >
+    <ResultCounts
+      :total-passed="results.totalPassed"
+      :total-failed="results.totalFailed"
+      :total-skipped="results.totalSkipped"
+      :total-pending="results.totalPending"
+      :order="['PASSED', 'FAILED', 'SKIPPED', 'PENDING']"
+    >
+      <template #prefix>
+        <div
+          class="text-red-400 font-semibold text-sm flex items-center border-r-1px px-2"
```

Comment:
This is not a full height border in figma. This is also visible on the first line
![Screen Shot 2022-11-17 at 4 47 26 PM](https://user-images.githubusercontent.com/25158820/202576549-5bda94d3-9d7a-4b48-823b-a203345174e5.png)